### PR TITLE
OSDOCS-9350-deepdive-2: Configuring ingress cluster traffic using a N…

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.12+

Issue:
(https://issues.redhat.com/browse/OSDOCS-9350)

Link to docs preview:
* [Configuring ingress cluster traffic using an Ingress Controller](https://79812--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-ingress-controller.html)
* [Configuring ingress cluster traffic using a load balancer](https://79812--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-load-balancer.html)
* [Configuring ingress cluster traffic using a NodePort](https://79812--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-nodeport.html)

- [ ] SME has approved this change.
- [ ] QE has approved this change.

Additional information:
* [SME doc](https://docs.google.com/document/d/13w_tqS-xFFFrb74jF_rcgApUuXwrBsjVaEQGY2mE0KI/edit#heading=h.nn1hdtqlfcrd)

### CODE REVIEW

**Comment by dfitzmau** [comment]
The deep dive doc mentions:

Screenshot from 2024-08-02 14-10-12


Do I need to state these commands in the NodePort "Creating a project and service" procedure. Also do the updates in this section also apply to the linked Configuring ingress cluster traffic using an Ingress Controller and Configuring ingress cluster traffic using a load balancer docs?

File: modules/nw-creating-project-and-service.adoc [hunk 1]
```diff
@@ -6,44 +6,59 @@
...
+$ oc new-app <service_name>
 ----
 
-. To verify that the service was created, run the following command:
+. To patch the created `NodePort` service as a service that selects an available port on each node in your cluster, enter the following command. The command does not require that you define a route, router, or ingress controller for a cluster.
 +
 [source,terminal]
 ----
-$ oc get svc -n myproject
+$ oc patch svc/<service_name> --type='merge' -p='{"spec":{"type":"NodePort"}}' -n <service_project_name>
```
Comment (CFields651) [inline modules/nw-creating-project-and-service.adoc:37]: the patching of this service with a node port does not belong here.  that is already covered in this section: https://79812--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-nodeport

Comment (CFields651) [reply modules/nw-creating-project-and-service.adoc:37]: so basically, everything in the 4.16 docs in chapter 28.3.x is okay as it stands: https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/networking/configuring-ingress-cluster-traffic#configuring-ingress-cluster-traffic-ingress-controller

**Comment by CFields651** [comment]
@dfitzmau we need to talk about this because our docs already have a section on configuring a node port at https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/networking/configuring-ingress-cluster-traffic#configuring-ingress-cluster-traffic-service-external-ip.

**Pull request closed without merging**